src/invalid.ts

```diff
@@ -0,0 +1,79 @@
+import {SignalRef} from 'vega';
+import {ScaleChannel} from './channel';
+import {Mark, MarkDef} from './mark';
+import {isObject} from 'vega-util';
+
+/**
+ * Mixins for Vega-Lite Spec's Mark Definiton (to add mark.invalid)
+ */
+export interface MarkInvalidMixins {
+  /**
+   * Invalid data mode for marks, which defines how the visualization should represent invalid values (`null` and `NaN` in continuous scales without defined output for invalid values) in the marks and their scale domains.
+   *
+   * -  `"filter"` — *Exclude* all invalid values from the visualization's *marks* and *scales*.
+   * For path marks (for line, area, trail), this option will create paths that connect valid points, as if the points with invalid values do not exist.
+   *
+   * - `"break-paths"` —
+   * Break path marks (for line, area, trail) at invalid values.  For non-path marks, this is equivalent to `"filter"`.
+   * All *scale* domains will *exclude* these filtered data points.
+   *
+   * - `"break-paths-keep-domains"` —
+   * Break paths (for line, area, trail) at invalid values.  Hide invalid values for non-path marks.
+   * All *scale* domains will *include* these filtered data points.
+   *
+   * - `"include"` or `null` —
+   * Include all data points in the marks and scale domains. Each scale will use the output for invalid values defined in `config.scale.invalid`
+   * or, if unspecified, by default invalid values will produce the same visual values as zero (if the scale includes zero) or the minimum value (if the scale does not include zero).
```

Comment:
this behavior sounds more like `replace`?

Reply:
"show at default position" that's super long for a property name.   I can rename to "show" though.